src/test/tests.ts

```diff
@@ -896,7 +893,15 @@ describe('Server', function () {
       //do nothing
     });
 
-    client.unsubscribe(subId);
+    setTimeout(() => {
+      // TODO
+      // Hagaico I had the need to add this timeout here
```

Comment:
what do you mean before subscribes completes?
why not just unsubscribe on callback (where do nothing is)?